src/search.c

```diff
@@ -655,9 +693,15 @@ static void do_search(struct manifest *MoM, char search_type, char *search_term)
 	if (!hit_count) {
 		fprintf(stderr, "Search term not found.\n");
 	}
+
+	// need recursive size regardless of num_results
 	apply_size_penalty(bundle_info);
-	sort_results();
-	print_final_results(scope);
+	if (num_results != INT_MAX) {
+		sort_results();
+	} else {
```

Comment:
Personally I would like the results sorted regardless.

Reply:
If you look in the `else` clause I do `results = list_sort(results, bundle_size_cmp);` which sorts by bundle size only.

Reply:
Thanks, I had the test in the ```if``` reversed in my mind.